docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,52 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve query plans by not generating redundant predicates. ({issue}`16520`)
+* Fix query failure when invoking the {func}`json_table` function. ({issue}`20122`)
+* Fix query hanging when referencing a row field with a [SQL routine](/routines) ({issue}`19997`).
+* Fix potential incorrect results when using the {func}`ST_Centroid` and
+  {func}`ST_Buffer` functions for tiny geometries (less than 1e-3). ({issue}`20237`)
+
+## Delta Lake connector
+
+* Add the `parquet_ignore_statistics` catalog session property. ({issue}`20228`)
+* Improve performance of queries with selective joins on partition columns. ({issue}`20261`)
+* Reduce number of requests made to AWS Glue when listing tables, schemas, or
+  functions. ({issue}`20189`)
+* Fix incorrect results when querying Parquet files containing page indexes when
+  the query has filters on multiple columns. ({issue}`20267`)
+
+## Hive connector
+
+* Allow reading 3-element unions when the sub-union is composed of compatible
+  types or nulls. ({issue}`20233`)
+* Reduce number of requests made to AWS Glue when listing tables, schemas, or
+  functions. ({issue}`20189`)
+* Fix incorrect results when querying Parquet files containing page indexes when
+  the query has filters on multiple columns. ({issue}`20267`)
+
+## Hudi connector
+
+* Add support for enforcing that a filter on a partition key must be present in
+  the query. This can be enabled by with the
+  ``hudi.query-partition-filter-required`` configuration property or the
+  ``query_partition_filter_required`` session property. ({issue}`19906`)
+* Fix incorrect results when querying Parquet files containing page indexes when
+  the query has filters on multiple columns. ({issue}`20267`)
+
+## Iceberg connector
+
+* Add the `parquet_ignore_statistics` catalog session property. ({issue}`20228`)
+* Improve performance of queries with selective joins on partition columns. ({issue}`20212`)
+* Reduce number of requests made to AWS Glue when listing tables, schemas, or
+  functions. ({issue}`20189`)
+* Fix potential loss of data when running multiple `INSERT` queries at the same
+  time ({issue}`20093`).
+* Fix incorrect results when providing a nonexistent namespace while listing
+  namespaces. ({issue}`19980`)
+
+## SQL Server connector
+
+* Fix incorrect results for old `DATETIMEOFFSET` values. ({issue}`16559`)
```

Comment:
what does "old" mean

Reply:
Looking at the linked issues down the rabbithole I find https://gist.github.com/coffeemug/6168031 .. so dates before 1400 is potentially the answer